siteconfig.json5

```diff
@@ -4,7 +4,7 @@
     hideQssSaveButton: false,
 
     // File containing default values which are used after resetting user settings
-    defaultValues: "C:\\Users\\vagrant\\AppData\\Roaming\\gpii\\defaultSettings.json5",
+    defaultValues: "gpii\\defaultSettings.json5",
```

Comment:
based on how you named this, I would think that it refers to the values itself rather than a filepath or a filename.

Reply:
oh sorry, I see that you're using this name in the siteconfig file, then I'd say it's okay.

Reply:
I can see your confusion, so I just added a couple of comments above, to make it a bit clearer.